libgeopmd/src/MSRIOGroup.hpp

```diff
@@ -16,15 +16,13 @@
 #include "geopm/json11.hpp"
 
 #include "geopm/Cpuid.hpp"
-#include "geopm_public.h"
 #include "geopm_time.h"
 
 #include "geopm/IOGroup.hpp"
 
 extern "C"
 {
-    int GEOPM_PUBLIC
-        geopm_allowlist(size_t result_max, char *result);
+    int geopm_allowlist(size_t result_max, char *result);
```

Comment:
Why were all these changes in MSRIOGroup.cpp necessary?

Reply:
MSRIOGroup.hpp was moved from public to private, but the public markings were left behind:

https://github.com/geopm/geopm/pull/3462

this is a fixup to that change ^^^